### ISSUE TITLE
salt-run manage.versions fails with AttributeError

### ISSUE THREAD

**Issue by Sakorah** [saltstack/salt#42660]
### Description of Issue/Question
Trying to run 'salt-run manage.versions' to get a list of my minions and see which version they are running. It fails with an AttributeError.

### Setup
Saltmater running on Ubuntu 14.04 with version 2017.7.0

### Steps to Reproduce Issue
```
user@salt-ext01:~$ sudo salt-run manage.versions
Exception occurred in runner manage.versions: Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/salt/client/mixins.py", line 392, in _low
    data['return'] = self.functions[fun](*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/salt/runners/manage.py", line 666, in versions
    minion_version = salt.version.SaltStackVersion.parse(minions[minion])
  File "/usr/lib/python2.7/dist-packages/salt/version.py", line 261, in parse
    if version_string.lower() in cls.LNAMES:
AttributeError: 'bool' object has no attribute 'lower'
```

### Versions Report
```
user@salt-ext01:~$ salt --versions-report
Salt Version:
           Salt: 2017.7.0

Dependency Versions:
           cffi: 0.9.2
       cherrypy: 3.2.2
       dateutil: 1.5
      docker-py: Not Installed
          gitdb: 0.5.4
      gitpython: 0.3.2 RC1
          ioflo: Not Installed
         Jinja2: 2.7.2
        libgit2: 0.22.2
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: 0.9.1
   msgpack-pure: Not Installed
 msgpack-python: 0.4.6
   mysql-python: 1.2.3
      pycparser: 2.10
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: 0.22.0
         Python: 2.7.6 (default, Oct 26 2016, 20:30:19)
   python-gnupg: Not Installed
         PyYAML: 3.10
          PyZMQ: 14.0.1
           RAET: Not Installed
          smmap: 0.8.2
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.0.5

System Versions:
           dist: Ubuntu 14.04 trusty
         locale: UTF-8
        machine: x86_64
        release: 4.4.0-87-generic
         system: Linux
        version: Ubuntu 14.04 trusty
```

**Comment by gtmanfred** [comment]
This is already fixed for 2017.7.1

https://github.com/saltstack/salt/pull/42436

**Comment by Sakorah** [comment]
Cool! I just searched the open issues not the closed ones - so this can be closed too!
